component/notification/src/main/java/org/exoplatform/social/notification/Utils.java

```diff
@@ -291,4 +297,8 @@ public static ActivityManager getActivityManager() {
   public static RelationshipManager getRelationshipManager() {
     return getService(RelationshipManager.class);
   }
+
+  public static boolean isNewsActivityNotificationsEnabled() {
+    return Boolean.parseBoolean(PropertyManager.getProperty(NEWS_ACTIVITY_NOTIFICATIONS_PROPERTY_NAME));
+  }
```

Comment:
Please update the PR based on the logic I proposed

```suggestion
  /**
   * Checks if the notification is enabled for activities of type activityType
   * @param activityType type of activity to check if their notification is enabled
   * @return true if the notification is enabled for this Activity Type
   */
  public static boolean isActivityNotificationsEnabled(String activityType) {
    String disabledNotifications = PropertyManager.getProperty(NEWS_ACTIVITY_NOTIFICATIONS_PROPERTY_NAME);
    if(StringUtils.isNotBlank(disabledNotifications)) {
      String[] activityTypes = disabledNotifications.split(",");
      return ! Arrays.asList(activityTypes).contains(activityType);
    }
    return true;
  }
```

Reply:
Okay